gen.go

```diff
@@ -0,0 +1,3 @@
+package main
+
+//go:generate go run github.com/99designs/gqlgen generate --verbose
```

Comment:
Using go generate, go will find this file and run the commands in it automatically.